### ISSUE TITLE
[a11y] In the "Select Language" Menu, some of the Buttons are missing a programatic label

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#12790]
**Describe the bug**
Issue: In the "Select Language" Menu, some of the Buttons are missing a programatic label.

Location: Find Programs Home Page
 
 Some languages use non-Latin characters like Arabic, Chinese, or Japanese, which could be why some of the languages are not announced.

**WCAG Success Criterion**
4.1.2 Name, Role, Value

**To Reproduce**
PC/JAWS
 1. Navigate to "Select Language"
 2. Open "Select Language" by pressing Enter or Space.
 3. Navigate the Language Menu Items with Up/Down Arrow Keys or Tab/Shift + Tab.
 
 iPhone/VO
 1. Navigate to the "Menu" Button.
 2. Open the "Menu" Button by performing a double-tap with 1 finger.
 3. Navigate to "Select Language" by swiping right.
 4. Open the "Select Language" Button by performing a double-tap with 1 finger.
 5. Navigate the Language Menu Items by swiping right/left. 
 
 Some languages use non-Latin characters like Arabic, Chinese, or Japanese, which could be why some of the languages are not announced.

**Expected behavior**
 Expectation: Each Language Button should have a clear and descriptive name.

Provide an accessible name.
 Ensure every <Button> element has a clear text label or an aria-label attribute.
 
 *For Traditional Chinese:
 <button aria-label="繁體中文" lang="zh-Hant">繁體中文</button>* 
 The aria-label ensures the name is read, and the lang="zh-Hant" tells JAWS to use a Chinese voice profile to pronounce it correctly.
 
 Apply the correct lang attribute to each button.
 *lang="es" for Spanish*
 
 Ensure the page uses UTF-8 encoding to properly render and read non-Latin scripts

**Comment by gwendolyngoetz** [comment]
We have `zh-TW` configured for Chinese currently. We should make sure if using `zh-Hant` doesn't result in confusion before making that change.

**Comment by rockycodes** [comment]
Charlotte also raised this as an issue for languages generally. Because they are missing the "lang" attribute, all languages will be read out in English, instead of in their native language.

Observation: Properly marking changes in natural language will allow users to render
languages appropriately across all technologies. For example, users who are reading the
document in another language will be able to substitute the appropriate control codes
(markup) where language changes occur to ensure that the braille translation software
will generate the correct characters (e.g. accented characters). These control codes also
prevent braille contractions from being generated, which could further confuse the
user. Braille contractions combine commonly used groups of characters that usually
appear in multiple cells into a single cell. For example, "ing" which usually takes up three
cells (one for each character) can be contracted into a single cell. Similarly, speech
synthesizers that "speak" multiple languages will be able to generate the text in the
appropriate accent with proper pronunciation. If changes are not marked, the
synthesizer will try to speak the words in the primary language it works in. Thus, the
French word for car, "voiture", would be pronounced "voter" by a speech synthesizer
that uses English as its primary language. Users who are unable to translate languages
themselves will be able to have unfamiliar languages translated by machine translators.
Remediation: Ensure that all inline changes of language are properly marked up using
the lang attribute.

WCAG Standards: [3.1.2 Language of parts](https://www.w3.org/TR/WCAG21/#language-of-parts)
